ironfish/src/primitives/target.ts

```diff
@@ -7,28 +7,25 @@ import { BigIntUtils } from '../utils/bigint'
 /**
  *  Minimum difficulty, which is equivalent to maximum target
  */
-const MIN_DIFFICULTY = BigInt(131072)
+const MIN_DIFFICULTY = 131072n
 
 /**
  * Maximum target, which is equivalent of minimum difficulty of 131072
  * target == 2**256 / difficulty
  */
-const MAX_TARGET = BigInt(
-  '883423532389192164791648750371459257913741948437809479060803100646309888',
-)
+const MAX_TARGET = 883423532389192164791648750371459257913741948437809479060803100646309888n
 
 /**
  *  Maximum number to represent a 256 bit number, which is 2**256 - 1
  */
-const MAX_256_BIT_NUM = BigInt(
-  '115792089237316195423570985008687907853269984665640564039457584007913129639935',
-)
+const MAX_256_BIT_NUM =
```

Comment:
Here do you want to do this for readability?
```javascript
const MAX_256_BIT_NUM = 2n ** 256n - 1n
```

Reply:
could do in another change too

Reply:
I think that's clean, but I'd recommend doing it in another PR just to avoid the back-and-forth